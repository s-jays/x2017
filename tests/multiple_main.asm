FUNC LABEL 4
    PRINT VAL 4
    RET
FUNC LABEL 5
    PRINT VAL 5
    RET
FUNC LABEL 0
    PRINT VAL 1
    RET
FUNC LABEL 0
    PRINT VAL 2
    RET
FUNC LABEL 0
    PRINT VAL 3
    RET
