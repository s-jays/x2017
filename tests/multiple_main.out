Program could not be executed: Did not have exactly one main()
